MDANSE_GUI/Src/MDANSE_GUI/MolecularViewer/MolecularViewer.py

```diff
@@ -1113,6 +1243,7 @@ def update_renderer(self):
         if self.reset_camera:
             self._renderer.ResetCamera()
             self.reset_camera = False
+            self._camera.SetWindowCenter(0.0, 0.0)
```

Comment:
Stupidly petty thing, feel free to ignore, ensure all work is done before setting flag:

```suggestion
            self._camera.SetWindowCenter(0.0, 0.0)
            self.reset_camera = False
```